docs/DesignDocs/AttestationApiRequirements.md

```diff
@@ -0,0 +1,291 @@
+Proposed Attestation API Requirements
+=====================================
+
+This design document proposes requirements for APIs that applications can
+use for performing attestation of enclave applications.
+
+Motivation
+----------
+
+Requirements:
```

Comment:
Add requirement "able to support platforms of different endianness"?

Reply:
Done